prompt/prompt-model/src/commonMain/kotlin/ai/koog/prompt/dsl/Prompt.kt

```diff
@@ -17,6 +17,7 @@ import kotlin.time.Duration
  * @property id The unique identifier for the prompt.
  * @property params The language model pa rameters associated with the prompt. Defaults to [LLMParams].
  */
+// FIXME move it from dsl package up to the module root package?
```

Comment:
This is not related directly to this PR, but I've noticed it while preparing it and want to raise a question. Currently, `Prompt` class is located in the `dsl` package, which seems kinda strange, since it's not about DSL, but rather a core structure of this whole module, a prompt model. So I think it's better to move it to the root module package or at least together with other models to `model`